mono/mini/aot-compiler.c

```diff
@@ -9042,12 +9042,18 @@ emit_llvm_file (MonoAotCompile *acfg)
 	if (acfg->aot_opts.llvm_only) {
 		/* Use the stock clang from xcode */
 		// FIXME: arch
-		command = g_strdup_printf ("clang++ -fexceptions -march=x86-64 -fpic -msse -msse2 -msse3 -msse4 -O2 -fno-optimize-sibling-calls -Wno-override-module -c -o \"%s\" \"%s.opt.bc\"", acfg->llvm_ofile, acfg->tmpbasename);
+		// Only output unsightly quotes for the unusual case of spaces in paths.
```

Comment:
Why does it hurt to always have the `"`? That seems like an unnecessary "optimization".

Reply:
It is ugly and unnecessarily verbose when viewing logs & copy/pasting their commands.

Reply:
Note that the code was already using wrap_path for similar reason, though it was leaking everything it returned.
Another idea is to use wrap_path, but it have it store all its results for freeing.